torchgen/gen_executorch.py

```diff
@@ -291,15 +291,28 @@ def gen_functions_declarations(
 def gen_headers(
     *,
     native_functions: Sequence[NativeFunction],
+    gen_custom_ops_header: bool,
```

Comment:
What's the overhead if we always generate CustomOpsNativeFunctions.h?

Reply:
Yeah I didn’t want to generate it if there’s no custom ops. Let me land this to unblock internal use case and then follow up.